src/components/LandingCard.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react"
+import { Link } from "gatsby"
+
+const LandingCard = (props) => {
```

Comment:
Can remove `content` assignment

```suggestion
const LandingCard = ({ content, idx }) => (<div className="landing-card" key={props.idx}>
```